### DISCUSSION TITLE
.exe asks to install an application from the store (which won't be found)

### DISCUSSION THREAD

**Discussion by MikeSlk** [Rem0o/FanControl.Releases#1958] [Q&A]
Hi, sorry I'm new to both Github and FanControl and a noob in IT so please apologies if question is stupid.

I've downloaded the zip, extracted it, launched the .exe ("application" type file right ?). A pop up pops up asking whether I want to search an app in the store, saying I need to install an app for this task. What am I doing wrong ?

Also, I have a brand new Matebook D16. Which version of V165 shall I use, as I 'm failing to see the difference ? 4.8 or 7.0 ?

Thanks and again apologies

**Comment by Rem0o** [comment]
That's probably because you downloaded the 7.0 which requires you to download the dotnet framework version 7.0 . However you got a laptop, so FanControl won't work on it anyway.

**Comment by MikeSlk** [comment]
Oh sorry, is FanControl only for Desktops ? Would there be an alternative (I used to use NBFC on my previous laptop, but stopped being updated since 2019) ?

**Reply by Rem0o** [reply]
Yup desktops only.
